Vagrantfile

```diff
@@ -1,19 +1,22 @@
-user     = ENV['NEXUS_USERNAME']
-pass     = ENV['NEXUS_PASSWORD']
-
 Vagrant.configure("2") do |config|
-  config.vm.box = "centos/7"
+  config.vm.box = "bento/ubuntu-20.04"
```

Comment:
any reason for not using ubuntu/focal64 ?

Reply:
the first one that popup on my google search =D